Update instructions to include Fork (#12712)
DataDog/integrations-core: docs/dev/new_check_howto.md

## Setup

> Fork* and Clone the [integrations-extras repository][3]. By default, that tooling expects you to be working in the `$HOME/dd/` directory—this is optional and can be adjusted with configuration later.

Reviewer: ```suggestion
**Fork** and clone the [integrations-extras repository][3]. By default, the tooling expects you to be working in the `$HOME/dd/` directory. This is optional and can be adjusted with configuration later.
```

```shell
mkdir $HOME/dd && cd $HOME/dd       # optional
git clone https://github.com/DataDog/integrations-extras.git
```

> **Note** A Fork is required if you intend on publishing the check or contributing to [integrations-extras repository][3].

Reviewer: ```suggestion
If you intend on publishing the check or contributing to [integrations-extras repository][3], you must create a fork.
```
